src/screens/SiteDetailScreen.js

```diff
@@ -257,6 +257,42 @@ class SiteDetailScreen extends Component {
         }
     };
 
+    renderAdminEditButton = () => {
+        const {navigation: {navigate}} = this.props;
+
+        return (
+            <Icon
+                reverse={true}
+                name='edit'
+                size={30}
+                color={navyBlueButton}
+                onPress={() => navigate(navKeys.EDIT_SITE)}
+            />
+        );
+
+    };
+
+    renderAdminOptions = () => {
+        const {sectionTitleStyle, adminOptionsButtonContainerStyle} = styles;
+        const {currentUser} = this.props;
+
+        if (!currentUser || !currentUser.isAdmin) {
+            return null;
+        }
+
+        return (
+            <View>
+                <Text style={sectionTitleStyle}>
+                    {admin_options}
+                </Text>
+
+                <View style={adminOptionsButtonContainerStyle}>
+                    {this.renderAdminEditButton()}
+                </View>
+            </View>
+        )
+    };
```

Comment:
Admin options in the `SiteDetailScreen` that only appear if the user is an Administrator. The Edit button will launch the `EditSiteScreen`.